config.gui.prod.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-	phaseControl: true,
+	phaseControl: false,
```

Comment:
why did you remove this?

Reply:
Yes I understand, but its false by default, you can enable it if you need it. I disabled it because most beamlines do not have phase control, so its like an additional option.